translations/tiled_ru.ts

```diff
@@ -560,72 +560,72 @@
     <message>
         <location filename="../src/tiled/mainwindow.ui" line="+49"/>
         <source>&amp;File</source>
-        <translation>&amp;Файл</translation>
+        <translation>Файл</translation>
```

Comment:
Why remove the accelerator?

Reply:
Please decline this PR, I will redo the translation and will make new PR.